pkg/v1/providers/infrastructure-vsphere/v1.1.0/ytt/overlay.yaml

```diff
@@ -226,6 +280,14 @@ spec:
           dhcp4: true
           #@ end
       numCPUs: #@ data.values.VSPHERE_WORKER_NUM_CPUS
+#@ if data.values.VSPHERE_WORKER_PCI_DEVICES != "" and data.values.VSPHERE_WORKER_PCI_DEVICES != None:
+      #@overlay/replace
+      pciDevices: #@ get_pci_devices(data.values.VSPHERE_WORKER_PCI_DEVICES, data.values.VSPHERE_IGNORE_PCI_DEVICES_ALLOW_LIST)
+#@ end
+#@ if data.values.VSPHERE_WORKER_CUSTOM_VMX_KEYS != "" and data.values.VSPHERE_WORKER_CUSTOM_VMX_KEYS != None:
+      #@overlay/replace
```

Comment:
```suggestion
      #@overlay/match missing_ok=True
```

Reply:
Hi @randomvariable  , We need to use "replace" for get_pci_devices() functionality to work . 
what is the need for using match ? Please can you explain this suggestion ?